sdcm/node_exporter_setup.py

```diff
@@ -42,3 +42,42 @@ def install(node: "BaseNode | None" = None, remoter: "Remoter | None" = None):
             systemctl enable node_exporter.service
             systemctl start node_exporter.service
         """))
+
+
+class SyslogNgExporterSetup:  # pylint: disable=too-few-public-methods
+    @staticmethod
+    def install(node: "BaseNode | None" = None, remoter: "Remoter | None" = None):
+        assert node or remoter, "node or remoter much be pass to this function"
+        if node:
+            node.install_package('wget')
+            remoter = node.remoter
+        remoter.sudo(shell_script_cmd("""
+            wget https://github.com/brandond/syslog_ng_exporter/releases/download/0.1.0/syslog_ng_exporter
```

Comment:
the only concern is. this code is not very maintained, and there's only one release of it.

Reply:
In case of problems we can write own exporter

Reply:
yeah it's not really that hard to rewrite it.
anyhow it's not a critical part, and we'll be able to live without it, if it would break for any reason.